packages/builder/src/pages/builder/portal/settings/ai/index.svelte

```diff
@@ -153,80 +154,83 @@
       aiConfig = (await API.getConfig(ConfigType.AI)) as AIConfig
       const licenseKeyResponse = await API.getLicenseKey()
       hasLicenseKey = licenseKeyResponse?.licenseKey
+      mounted = true
```

Comment:
What would happen if there is an error getting this? The full section will not be rendered. also, this might cause confusion as the mounted will be false for mounted components that failed. It should either be moved outside the try/catch or changed to `aiConfig` loaded or similar. On the other hand, if it's the latest, it could be as easy to have the check to be `if(aiConfig && licenseKey)`